gulpfile.js

```diff
@@ -46,6 +46,8 @@ var paths = {
 			'./browser/scripts/worldEditor/worldEditorOriginGrid.js',
 			'./browser/scripts/worldEditor/worldEditorRadialHelper.js',
 
+			'./browser/scripts/worldEditor/objectPlacementHelper.js',
```

Comment:
just a note: these files don't need to be here for the player. let's separate this into an editor list soon